10-eos-storage.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: First steps in LHCb
+subtitle: Getting your data onto EOS
+minutes: 10
+---
+
+> ## Learning Objectives {.objectives}
+>
+> * Run a ganga job which puts output onto EOS
+> * Open and view the files on EOS
+
+Because you have only 100 Gb of AFS workspace, you will need to put most large
+data files and ntuples somewhere else. Unless your own institute has a large computing
+cluster and runs LHCb software locally, the most convenient place is EOS. You get
+2 Tb of space which is allocated to you, and this is disk space so you do not
+have to wait for files to be staged or replicated. 
+
+Note that this tutorial aims to teach you a workflow which is particularly convenient
+for getting your data accessible to interactive analysis on lxplus, or analysis on lxbatch.
+It is perfectly possible to have a different workflow in which all data goes to
+the grid. You could then do all your work through ganga, although you cannot work interactively
```

Comment:
I don't agree actually : I tried to write this in a pedagogical way which explains to the student why they are being taught this and why it is NOT the one true way, because I don't like creating zombie armies of the misinformed. ;-) So I prefer expanding on this rather than removing it.